package.json

```diff
@@ -93,6 +93,7 @@
     "@ungap/from-entries": "^0.2.1",
     "ansi-styles": "^5.0.0",
     "ascii-table": "0.0.9",
+    "better-opn": "^3.0.0",
```

Comment:
We'll need to use `v2` since `v3` requires Node.js 12

Reply:
Good catch! Done in https://github.com/netlify/cli/pull/2169/commits/07383eb749d05cf9ee220e009fbeb9cb96d94f2e.